client/containers/Pub/SpubHeader/SpubHeaderToolBar/SpubHeaderToolbar.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { Dispatch, SetStateAction, useMemo } from 'react';
+import { Tab, Tabs, TabId, Icon, IconName, Button } from '@blueprintjs/core';
+import Color from 'color';
+
+import { GridWrapper } from 'components';
+import { SubmissionStatus } from 'types';
+import { usePageContext, usePendingChanges } from 'utils/hooks';
+
+require('./spubHeaderToolbar.scss');
+
+const renderTabTitle = (icon: IconName, title: string) => (
+	<>
+		<Icon icon={icon} /> {title}
+	</>
+);
+
+type Props = {
+	selectedTab: TabId;
+	onSelectTab: Dispatch<SetStateAction<TabId>>;
+	status: SubmissionStatus;
+	showSubmitButton: boolean;
+	onSubmit: () => any;
```

Comment:
I'm dying on a hill about `unknown` vs `any` [over here](https://github.com/pubpub/pubpub/pull/1866#discussion_r827360781) in Ben's PR.

Reply:
![image](https://user-images.githubusercontent.com/34730449/158492336-5ca0c22a-76be-4da5-9b10-b47c850ec186.png)

not if i can help it